guacamole/src/main/webapp/translations/nl.json

```diff
@@ -1,6 +1,6 @@
 {
     
-    "NAME" : "Nederlands",
+    "NAME" : "Nederlands (NL)",
```

Comment:
I translated from the English json file, which had the country code added at that time. I assumed that that was correct. It has been removed from the English json file since then.